pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -75,6 +75,9 @@ spec:
             {{ if .CephFSKernelMountOptions }}
             - "--kernelmountoptions={{ .CephFSKernelMountOptions }}"
             {{ end }}
+            {{ if .EnableCSIAddonsSideCar }}
```

Comment:
Is the sidecar needed in both the cephfs provisioner and the plugin? If it's currently only for the fencing, perhaps it's only needed in the provisioner?

Reply:
Yes, that's correct.
Thanks Travis, updated. 
PTAL.